### PR TITLE
Nginx self-signed certificate creation

### PR DESCRIPTION
**Pull request by andytson-inviqa**
depends on #61 and #62

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
I'd thought of adding optional env var cert/key data injection as well, but think this is enough for an iterative improvement

File: php-nginx/README.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ docker-compose build php56_nginx
 docker-compose push php56_nginx
 ```
 
+## About
+
+This is a Docker image for PHP (using the php-fpm SAPI) and Nginx. It uses, by
+default, our recommended default configuration, including:
+
+* A default HTTPS only website, with HTTP redirecting to HTTPS
```
Comment (sroze) [inline php-nginx/README.md]: AFAIK this PR do not introduce a HTTP to HTTPS redirection (which shouldn't IMHO), right?

Comment (andytson-inviqa) [inline php-nginx/README.md]: It should for new projects that want to gain from security and performance improvements. The default state is that HTTP redirects if HTTPS is enabled, but HTTP can be enabled as well

Comment (andytson-inviqa) [inline php-nginx/README.md]: This should be considered both for environments that don't have an SSL LB terminator, and those where the security requirements ask that inter-container traffic is TLS encrypted. Getting Vault to manage the TLS certs will help make that an easy reality for any project that choses to once the groundwork is done.

Comment (sroze) [inline php-nginx/README.md]: Agree with you, we should add a bold recommendation but not force that by default, as it wouldn't help newcomers... :/

Comment (sroze) [inline php-nginx/README.md]: > The default state is that HTTP redirects if HTTPS is enabled, but HTTP can be enabled as well

Where is that in your PR? 🤔

Comment (andytson-inviqa) [inline php-nginx/README.md]: It was introduced in a previous PR, that PR supported HTTPS as well, just was disabled by default until at least self signed certs were available.

#61 https://github.com/continuouspipe/dockerfiles/pull/61/files#diff-079ac36fbe458f15bd03fc001098e215R44

Comment (andytson-inviqa) [inline php-nginx/README.md]: https://github.com/continuouspipe/dockerfiles/pull/61/files#diff-17c9a37da5a06bfcb5ccfe32f06a3258R5 for the actual place this happens

Comment (sroze) [inline php-nginx/README.md]: Alright, so it was already merged so I believe that @kierenevans & you were okay with that. I'm not convinced at all by the UX but we'll see :)

Comment (andytson-inviqa) [inline php-nginx/README.md]: Can you elaborate that somewhere when you have the tim?

**Pull request merged**